_docs/privacy/no-tracking.md

```diff
@@ -4,16 +4,4 @@ category: Privacy
 order: 65
 ---
 
-<p>
-    DuckDuckGo does not store any personal information, e.g. IP addresses or user
-    agents: see
-    <a href="https://duckduckgo.com/privacy">our privacy policy</a> for details.
-    In addition to <a href="https://duckduckgo.com/">our default version</a>, we
-    also offer two non-JavaScript versions (<a href="https://duckduckgo.com/html">HTML</a>
-    &amp; <a href="https://duckduckgo.com/lite">lite</a>), a
-    <a href="https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/">Tor onion service</a>, several
-    <a href="https://duckduckgo.com/settings">privacy settings</a> (including POST
-    &amp; RefControl), and we allow you to use
-    <a href="https://duckduckgo.com/params">URL parameters</a> instead of cookies
-    for our Cloud Save feature to store settings.
-</p>
+DuckDuckGo never tracks you, per [our privacy policy](https://duckduckgo.com/privacy). In addition to [our default version](https://duckduckgo.com/), we also offer two non-JavaScript versions ([HTML](https://duckduckgo.com/html) & [lite](https://duckduckgo.com/lite)), a [Tor onion service](https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/), several [privacy settings](https://duckduckgo.com/settings) (including POST & RefControl), and we allow you to use [URL parameters](https://duckduckgo.com/params) instead of cookies for our Cloud Save feature to store settings.
```

Comment:
Updated copy here

Reply:
One tiny suggestion – could say "our anonymous Cloud Save feature" here, to clarify that it's not connected with an email address or other personal info.

Reply:
Thanks @tagawa, I'll pass this along to the Comms team